packages/server-dom-esbuild/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@ml-in-barcelona/server-dom-esbuild",
+  "description": "React Server Components bindings for DOM using Esbuild. This is intended to be integrated into meta-frameworks. It is not intended to be imported directly.",
+  "version": "0.0.0",
+  "exports": {
+    ".": "./dist/ReactServerDOMEsbuild.js"
+  },
+  "keywords": [
+    "reason-react",
+    "server-reason-react",
+    "esbuild"
+  ],
+  "homepage": "https://github.com/ml-in-barcelona/server-reason-react",
+  "bugs": "https://github.com/ml-in-barcelona/server-reason-react/issues",
+  "scripts": {
+    "build": "esbuild ReactServerDOMEsbuild.js --bundle --format=esm --outfile=dist/ReactServerDOMEsbuild.js",
+    "release": "npm run build && npm publish"
```

Comment:
no need to add scripts (the building happens at the consumer) so the demo does it and monorepo will do it